app.js

```diff
@@ -1,50 +1,106 @@
-let bookList = [];
-
-function displayBooks() {
-  const outputFiled = document.getElementById('displayField');
-  outputFiled.innerHTML = '';
-
-  bookList.forEach((book) => {
+class bookBank {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+// add books to store
+class bookStore {
+  // add method to get book from store
+  static getbookBank() {
+    // check if local storage is empty
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      // otherwise get books from store
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+  // create method to add book to storage
+  static addBook(book) {
+    const books = bookStore.getbookBank();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+  // remove book from storage
+  static removeBook(title) {
+    const books = bookStore.getbookBank();
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
```

Comment:
- [ ] [OPTIONAL] It is more advisable to use array methods that don't mutate the array. In this case, please should consider using `books.slice()` instead of `books.splice()` 🙏